source/connect/client.txt

```diff
@@ -0,0 +1,114 @@
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``mongocxx::client`` object
+to connect to MongoDB.
+
+.. _cpp-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+       For more information about the ``authSource`` connection option, see
+       :ref:`cpp-auth`. 
+
+   * - ``host[:port]``
+
+     - Required. The host and optional port number where MongoDB is running. If you don't
+       include the port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. See
+       :ref:`cpp-connection-options` for a full description of
+       these options.
+
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string>` in the
+MongoDB Server documentation.
+
+Create a ``mongocxx::client``
+-----------------------------
+
+To create a connection to MongoDB, construct an instance of the ``mongocxx::uri`` class,
+passing the connection URI as a string to the constructor. Then, pass the instance of
+the ``mongocxx::uri`` class to the ``mongocxx::client`` constructor.
+
+In the following example, the driver uses a sample connection URI to connect to a MongoDB 
+deployment on port ``27017`` of ``localhost``: 
+
+.. literalinclude:: /includes/connect.cpp
+      :language: c++
+      :copyable: true
+      :start-after: // start-client
+      :end-before: // end-client
+      :emphasize-lines: 2-3,6-7
+
+.. tip:: Reuse Your Client
+
+   Because each ``mongocxx::client`` object represents a pool of connections to the
+   database, most applications require only a single instance of
+   ``mongocxx::client``, even across multiple requests. 
+
+API Documentation
+-----------------
+
+To learn more about creating a ``mongocxx::client`` object in {+driver-short+},
+see the following API documentation:
+
+- `mongocxx::client <{+api+}/current/classmongocxx_1_1v__noabi_1_1client.html>`__ 
```

Comment:
This link is 404ing. Looks like "/current" might be duplicated?

Reply:
good catch